nightingale_interface/screens/face_screen.py

```diff
@@ -0,0 +1,45 @@
+from kivy.uix.screenmanager import Screen
+from kivy.uix.image import Image
+
+from kivymd.uix.label import MDLabel
+from kivy.uix.button import Button
+from kivy.uix.label import Label
+
+from kivy.uix.screenmanager import NoTransition
+from screens.screen_config import ScreenConfig as cfg
+
+
+class FaceScreen:
+    face_name = "facescreen"
+
+    def to_homescreen(self, button_data):
+        button_data.parent.manager.transition = NoTransition()
+        cfg.LAST_SCREEN = self.face_name
+        button_data.parent.manager.current = (
+            "homescreen"  # Should just be the "Home" screen
+        )
+
+    def face_build(self):
+        screen = Screen(name=self.face_name)
+
+        current_action_label = Label(
+            text=cfg.CURRENT_ACTION,
+            # font_style="H4",
+            halign="center",
+            pos_hint={"center_x": 0.5, "center_y": 0.9},
+        )
+        current_action_label.font_size = "50sp"
+
+        screen.add_widget(current_action_label)
+
+        # remove press color animation
+        screen.add_widget(
+            Button(
+                background_normal="images/Face.png",
```

Comment:
Face.png should be in white if the background is black. I'm open to changing the color theme on the app to white though which might look better overall

Reply:
I was thinking that white would feel more "sterile" and might fit better for the environment but black probably feels sleeker. Do you feel that black is a bit hard to read? I think the black works well for the robot theme and we should keep it unless it's hard to read.